README.md

```diff
@@ -93,101 +63,47 @@ After you're finished please remove all the comments and instructions!
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-<!-- LIVE DEMO -->
 
 ## 🚀 Live Demo <a name="live-demo"></a>
 
-> Add a link to your deployed project.
 
 - [Live Demo Link](will be added later)
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
```

Comment:
- [ ] It would be nice if you remove the Live Demo section since you don't have any working live link 🤔 so it would be great if you remove the live demo to make your readme descriptive and professional 👍